.packit.yaml

```diff
@@ -0,0 +1,32 @@
+downstream_package_name: openscap
+jobs:
+- job: copr_build
+  metadata:
+    targets:
+    - fedora-all-x86_64
+  trigger: pull_request
+- job: tests
+  metadata:
+    targets:
+    - fedora-all-x86_64
+  trigger: pull_request
+- job: propose_downstream
+  metadata:
+    dist-git-branch: fedora-all
+  trigger: release
+specfile_path: openscap.spec
+synced_files:
+- .packit.yaml
+upstream_package_name: openscap
+actions:
+  post-upstream-clone:
+    - "wget -N https://src.fedoraproject.org/rpms/openscap/raw/rawhide/f/openscap.spec"
+    - "sed -i '/^Patch/d' openscap.spec"
+    - "sed -i '23 a BuildRequires: xmlsec1-devel xmlsec1-openssl-devel' openscap.spec"
+    - "sed -i '24 a Requires: xmlsec1 xmlsec1-openssl' openscap.spec"
+    # use git_am instead of patch since packit patches are generated with git-format-patch
```

Comment:
```suggestion
    - "sed -E -i 's/%cmake (.+)$/%cmake -S \"%{_vpath_srcdir}\" -B \"%{__cmake_builddir}\" \\1/'"
    # use git_am instead of patch since packit patches are generated with git-format-patch
```

Reply:
This won't work. I have tried to come up with a spec file without "if Fedora 32 then ... else" blocks downstream, as features like this shouldn't be delivered by means of monkey-patching.
It is possible to make `cmake` aware of the actual build directory (by passing it `-B %_vpath_builddir` that would be defined before, `__cmake_builddir` is not set correctly), but that would cause problems with the `cmake_build` macro.

Therefore, I propose to abandon efforts to please F32, as it will be irrelevant next month due to F34 being released.

Reply:
Hmm, well the right thing to do for Fedora 32 is to `sed -i '127 a %undefine __cmake_in_source_build' openscap.spec` as I did in the latest commit.

Reply:
You have to undefine __cmake_in_source_build in order to make out-of-source cmake-related macros to work. In F33+ is is done by default.